config/collector.yaml

```diff
@@ -119,21 +193,27 @@ processors:
       - set(attributes["top_level_controller_type"], "CronJob") where resource.attributes["k8s.cronjob.name"] != nil
       - set(attributes["top_level_controller_name"], resource.attributes["k8s.cronjob.name"]) where resource.attributes["k8s.cronjob.name"] != nil
 
-receivers:
-  otlp:
-    protocols:
-      grpc:
-        endpoint: ${env:MY_POD_IP}:4317
-      http:
-        cors:
-          allowed_origins:
-          - http://*
-          - https://*
-        endpoint: ${env:MY_POD_IP}:4318
-  otlp/self-metrics:
-    protocols:
-      grpc:
-        endpoint: ${env:MY_POD_IP}:14317
+exporters:
+  # The googlecloud exporter will export telemetry to different
+  # Google Cloud services:
+  # Logs -> Cloud Logging
+  # Metrics -> Cloud Monitoring
+  # Traces -> Cloud Trace
+  #
+  # Docs:
+  # https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter
+  googlecloud:
+    log:
+      default_log_name: opentelemetry-collector
+    user_agent: Google-Cloud-OTLP manifests:0.2.0 OpenTelemetry Collector Built By Google/0.121.0 (linux/amd64)
+
+  # The googlemanagedprometheus exporter will send metrics to
+  # Google Managed Service for Prometheus.
+  #
+  # Docs:
+  # https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlemanagedprometheusexporter
+  googlemanagedprometheus:
```

Comment:
does this need the IntToDouble flag somewhere

Reply:
It is turned on as part of the Google-Built OpenTelemetry Collector image's `ENTRYPOINT`. The way these manifests interact with the container as well as all other documented sources, the flag will always be on.

Reply:
Should the information about `intToDouble` flag be mentioned here to help guide in case of issues due to that flag ?

Reply:
Addressed in [9099496](https://github.com/GoogleCloudPlatform/otlp-k8s-ingest/pull/52/commits/9099496deee9f692f805fa6cf7140c74ec03602d)